lib/worker/src/domain/cache_path.rs

```diff
@@ -0,0 +1,14 @@
+#[derive(Debug)]
+pub struct CachePath(String);
+
+impl CachePath {
```

Comment:
should we use the `From` [trait](https://doc.rust-lang.org/std/convert/trait.From.html) here, like `AsRef` is used below?

Reply:
oh i see, `TryFrom` is implemented later